### PR TITLE
alternate language and CTA for renewal

### PR DESCRIPTION
**Pull request by zbtirrell**
See https://central.tri.be/issues/45629

### CODE REVIEW

File: src/Tribe/PUE/Checker.php [hunk 1]
```diff
@@ -564,16 +564,11 @@ public function display_license_error_message() {
...
-					'This license key is expired! You need to %1$srenew your license %2$s %3$s to get access to the latest versions including bug fixes, security updates, and new features.',
-					'tribe-common'
-				),
-				'<a href="http://m.tri.be/195y" target="_blank">',
-				$helper_text,
-				'</a>'
-			);
+			$expired_message = '<a href="http://m.tri.be/195y" target="_blank" class="button button-primary">' . __( 'Renew Your License Now', 'tribe-common' ) . '</a>';
+
+			$expired_message .= '<span class="screen-reader-text">' . __( ' (opens in a new window)', 'tribe-common' ) . '</span>';
```
Comment (borkweb) [inline src/Tribe/PUE/Checker.php]: The intent is to have the `$expired_message` span _inside_ the anchor tag, I believe. @trishasalas can correct me if I'm wrong.

Comment (trishasalas) [inline src/Tribe/PUE/Checker.php]: That is correct, see https://make.wordpress.org/accessibility/handbook/for-theme-developers/keeping-links-accessible/#new-windows-and-tabs

Comment (zbtirrell) [inline src/Tribe/PUE/Checker.php]: Ahh, I missed that, thanks!

**Comment by borkweb** [comment]
:+1:

**Pull request merged**